### DISCUSSION TITLE
clip gradient

### DISCUSSION THREAD

**Discussion by inveterate-cladocerus** [JavierBZ/Tarea3-IIC3697#28] [Ideas]
Hola, leí que en el artículo del ddqn hacían clip de los gradientes y también multiplicaban el gradiente combinado que se propagaba desde la fc advantage y fc value a la última capa convolucional por un factor, lo que mejoraba la estabilidad del entrenamiento. ¿Se puede probar esta técnica? ¿Alguien que lo haya implementado lo recomienda?
Saludos.

**Comment by JavierBZ** [comment]
Hola, ningún problema con que incluyan en el entrenamiento este tipo de técnicas. DQN sufre de inestabilidad que se ve reflejada en la práctica en problemas con el gradiente (exploding o vanishing gradients). 

En esta tarea la manera de estabilizar DQN es a través del uso de ReplayMemory y con el uso de red target. Si quieren agregar otras como las que se describen en el paper de Dueling DQN son bienvenidas, no les dará un beneficio de nota, pero si les hará más rápido el entrenamiento.
